tests/sqllogictests/suites/ee/06_ee_stream/06_0002_txn_stream.test

```diff
@@ -116,3 +116,187 @@ SELECT * FROM t_consume_append_only_6; -- empty
 
 statement ok
 drop database test_txn_stream;
+
+# multi table INSERT
```

Comment:
Maybe move the multi table insert  tests  into a new test file is better?

Reply:
Good idea. Done.